### ISSUE TITLE
Exception is thrown with the exmple private-claims.cpp

### ISSUE THREAD

**Issue by gggamemaster** [Thalhammer/jwt-cpp#351]
### What's your question?

it raise a exception

### Additional Context

the other example，i can run its well except example\partial-claim-verifier.cpp，because i do not install nlohmann-json。and why private-claims.cpp run fail?
![image](https://github.com/Thalhammer/jwt-cpp/assets/25485742/807aa727-72e7-47fb-9bea-e0728e63027b)
![image](https://github.com/Thalhammer/jwt-cpp/assets/25485742/bc9c7445-d77b-424e-a082-d8ad90f219d3)

**Comment by prince-chrismc** [comment]
What message does the exception print? 

They were not meant to be running a debugger. They are likely just expiring 

https://github.com/Thalhammer/jwt-cpp/blob/0d03bc5bc8b0fa16d535ec89e0dd6043b65b54d9/example/private-claims.cpp#L25

You should also check the call stack to see which verification function is failing, it's another way then reading the messages.

**Comment by gggamemaster** [comment]
![image](https://github.com/Thalhammer/jwt-cpp/assets/25485742/8f7af342-831b-40e7-9c0b-b53a5262ecda)
it seems time Expire?
![89dd4fa218574dffc765dcc5fd0379e](https://github.com/Thalhammer/jwt-cpp/assets/25485742/b22dd454-b0a4-4732-bdfb-f764ecf7b8f9)

**Comment by gggamemaster** [comment]
.set_expires_at(time + sec{15} + min{2})  menas that the token will time expire after 2 min 15 sec???just few time the exception araise..

**Comment by gggamemaster** [comment]
@Thalhammer

**Comment by prince-chrismc** [comment]
I'ts very difficult to see pictures, please keep the screenshot for the IDE. It's hard to follow with your language settings. It be great if you changed it to English (or french or german).

You can add 

https://github.com/Thalhammer/jwt-cpp/blob/0d03bc5bc8b0fa16d535ec89e0dd6043b65b54d9/example/partial-claim-verifier.cpp#L89-L92

And this will print a message that will make ot easier to know what error you are getting.

**Comment by gggamemaster** [comment]
you are right ，print "token expired" after i add the code on windows environment.

**Comment by gggamemaster** [comment]
i run private-claims.cpp fail in vs2022 debug 64 and vs2017 relaese64

**Comment by Thalhammer** [comment]
Thats pretty odd, ngl. The example sets a expiration of 1h and there should be no way he is halting the program for more than an hour (except on purpose) to debug it.

The example seems to [work fine on godbolt](https://godbolt.org/z/j1vq6bv6x) (or at least it did with the 0.7 release, I don't add master to godbolt).

**Comment by prince-chrismc** [comment]
The examples built last week with #348 so I dont think anything broke. Might be some localization with the time?

**Comment by prince-chrismc** [comment]
Oh that example is not ran in CI... 😡 

https://github.com/Thalhammer/jwt-cpp/blob/0d03bc5bc8b0fa16d535ec89e0dd6043b65b54d9/example/private-claims.cpp#L24

Is wrong that should be minus 15 seconds not plus.
